SerialPrograms/Source/PokemonSV/Inference/Overworld/PokemonSV_DirectionDetector.cpp

```diff
@@ -71,7 +75,16 @@ std::pair<double, double> DirectionDetector::locate_north(const ImageViewRGB32&
     const size_t min_size = size_t(screen_rel_size * screen_rel_size * min_object_size);
 
     std::pair<double, double> north_location(0, 0);
-    std::pair<double, double> zero_coord(0.9065, 0.8335);
+    OverworldDetector detector;
+    std::pair<double, double> zero_coord = detector.locate_ball(screen); // 0.9061, 0.8328
+    if (zero_coord.first > 0.91 || zero_coord.first < 0.90 || 
+        zero_coord.second > 0.84 || zero_coord.second < 0.82)
+    {
+        console.log("Unable to locate the overworld radar ball, falling back on hard coded location.");
```

Comment:
Make this log line either red or orange in color so it's easier to see when scrolling through.